helm/hpcc/templates/_helpers.tpl

```diff
@@ -1081,7 +1086,7 @@ Pass in dict with root, me and dali if container in dali pod
           {{ include "hpcc.configArg" . }},
 {{- end }}
           "--service={{ .me.name }}",
-{{ include "hpcc.daliArg" .root | indent 10 }}
+{{ include "hpcc.daliArg" (dict "root" .root "component" "Sasha" "optional" false) | indent 10 }}
```

Comment:
personal: I might have abbreviated this and only passed in optional if true, and default to false, since all bar 1 case is false